backend/app/router/endpoints/questions.py

```
@@ -2,48 +2,80 @@
-        )
-    ])
+    load_dotenv(dotenv_path=os.path.join(os.path.dirname(os.path.dirname(os.path.dirname(os.path.abspath(__file__)))), '.env'))
+
+    client = OpenAI(
+    api_key=os.environ.get("LLAMA_API_KEY"), 
+    base_url="https://api.llama.com/compat/v1/"
+)
+    
+    # Build the path to the markdown file dynamically
+    notes_path = os.path.join(
+        os.path.dirname(os.path.dirname(os.path.dirname(os.path.dirname(os.path.abspath(__file__))))),
```

Pylint (3.3.7)

[convention] Line too long (102/100)

(C0301)